.github/workflows/pull-request/review.yml

```
@@ -0,0 +1,121 @@
+name: Pull Request Automation
+
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+
+jobs:
+  build-design-system:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)